soc/wch/ch32v/qingke_v4b/Kconfig.defconfig

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2025 MASSDRIVER EI (massdriver.space)
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_QINGKE_V4B
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default $(dt_node_int_prop_int,/cpus/cpu@0,clock-frequency)
+
+config MAIN_STACK_SIZE
```

Comment:
You can probably drop these. They were added for the CH32V003 as it has 2 KiB of RAM.

Reply:
And you were 100 % right, hit this in a sample, so removed it before i forget.